python/tests/aggregate_tests/test_varchar_count_col.py

```diff
@@ -0,0 +1,61 @@
+from .aggtst_base import TstView
+
+
+class aggtst_varchar_count_col(TstView):
+    def __init__(self):
+        # Validated on Postgres
+        self.data = [{'c1': 3, 'c2': 4}]
+        self.sql = """CREATE MATERIALIZED VIEW varchar_count_col AS SELECT
+                      COUNT(c1) AS c1, COUNT(c2) AS c2
+                      FROM varchar_tbl"""
+
+
+class aggtst_varchar_count_col_groupby(TstView):
+    def __init__(self):
+        # Validated on Postgres
+        self.data = [{'id': 0, 'c1': 1, 'c2': 2},
+                     {'id': 1, 'c1': 2, 'c2': 2}]
+        self.sql = """CREATE MATERIALIZED VIEW varchar_count_col_gby AS SELECT
+                      id, COUNT(c1) AS c1, COUNT(c2) AS c2
+                      FROM varchar_tbl
+                      GROUP BY id"""
+                      
+                      
+class aggtst_varchar_count_col_distinct(TstView):
```

Comment:
this is the most important new case here, does the table have identical strings?
otherwise DISTINCT does nothing.

Reply:
Yes, the table contains two identical strings "hello".